src/base/bittorrent/session.cpp

```diff
@@ -373,6 +373,7 @@ Session::Session(QObject *parent)
     , m_announceToAllTrackers(BITTORRENT_SESSION_KEY("AnnounceToAllTrackers"), false)
     , m_announceToAllTiers(BITTORRENT_SESSION_KEY("AnnounceToAllTiers"), true)
     , m_asyncIOThreads(BITTORRENT_SESSION_KEY("AsyncIOThreadsCount"), 4)
+    , m_stop_tracker_timeout(BITTORRENT_SESSION_KEY("Stop_Tracker_timeout"), 3)    
```

Comment:
Don't sneak a default value change here. Leave the default value the same as it is currently, `1`, and wait for more feedback on https://github.com/qbittorrent/qBittorrent/pull/11807. Or at least wait for @Chocobo1 and @glassez feedback

Reply:
Also, name should be in camelCase like so: `m_stopTrackerTimeout`

Reply:
Also BITTORRENT_SESSION_KEY("**StopTrackerTimeout**")